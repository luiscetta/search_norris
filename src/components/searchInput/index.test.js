import React from 'react';

import { create } from 'react-test-renderer';

import SearchInput from './';


describe('SearchInput component', () => {
    describe('When the component is rendered', () => {

        it('Should match snapshot', () => {
            const wrapper = create(<SearchInput />);
            expect(wrapper.toJSON()).toMatchSnapshot();
        });
    })
});