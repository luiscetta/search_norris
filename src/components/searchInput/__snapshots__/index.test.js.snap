// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput component When the component is rendered Should match snapshot 1`] = `
<div
  className="input-field"
>
  <label
    htmlFor="input"
  >
    Search for a word (or text)
  </label>
  <input
    autoFocus={true}
    className="search-input"
    type="search"
  />
</div>
`;
